src/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+require('./dataBase/mongoose')
+const User = require('./models/user')
+const userRouter = require('./routes/user')
+
+
+const app = express()
+app.use(express.json())
+app.use(userRouter)
+
+
+module.exports = app
```

Comment:
Merge changes here with index.js and keep index.js. 
If your test fails without app.js , you need to update the test and we can look at that later.